### PR TITLE
Dna 7.0 beta

### PR DESCRIPTION
**Pull request by GarthDB**
## How and where has this been tested?
 - **How this was tested:** 
 - **Browser(s) and OS(s) this was tested with:** 

## To-do list

- [ ] If my change impacts other components, I have tested to make sure they don't break.
- [ ] If my change impacts documentation, I have updated the documentation accordingly.
- [ ] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).

- [ ] This pull request is ready to merge.

### CODE REVIEW

**Comment by GarthDB** [comment]
First pass fixing broken vars:

- [ ] accordion
- [ ] actionbutton
- [ ] alertbanner
- [ ] avatar
- [ ] badge
- [ ] body
- [ ] bottomnavigation
- [ ] breadcrumb
- [ ] button
- [ ] calendar
- [ ] card
- [ ] checkbox
- [ ] clearbutton
- [ ] coachmark
- [ ] code
- [ ] colorarea
- [ ] colorhandle
- [ ] colorloupe
- [ ] colorslider
- [ ] colorwheel
- [ ] column
- [ ] cyclebutton
- [ ] detail
- [ ] dial
- [ ] dialog
- [ ] drag
- [ ] dropzone
- [ ] fieldlabel
- [ ] heading
- [ ] helptext
- [ ] illustratedmessage
- [ ] inlinealert
- [ ] label
- [ ] link
- [ ] listitem
- [ ] logicbutton
- [ ] meter
- [ ] millercolumn
- [ ] panel
- [ ] picker
- [ ] popover
- [ ] progressbar
- [ ] progresscircle-indeterminate
- [ ] progresscircle
- [ ] quickactions
- [ ] radio
- [ ] rating
- [ ] scrollbar
- [ ] search
- [ ] searchwithin
- [ ] sidenav
- [ ] slider
- [ ] statuslight
- [ ] steplist
- [ ] switch
- [ ] tabbar
- [ ] table
- [ ] tabs
- [ ] tag
- [ ] taggroup
- [ ] textarea
- [ ] textfield
- [ ] thumbnail
- [ ] toast
- [ ] tooltip
- [ ] tray
- [ ] treeview
- [ ] well

File: components/badge/metadata/badge.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: Badge
+SpectrumSiteSlug: https://spectrum.adobe.com/page/badge/
+examples:
+  - id: badge
+    name: Standard
+    markup: |
+      <span class="spectrum-Badge spectrum-Badge--grey">Grey Badge</span>
```
Comment (lazd) [inline components/badge/metadata/badge.yml:7]: Oops, looks like this markup is old

File: components/badge/metadata/badge.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  - id: badge
+    name: Activity badges
+    markup: |
+      <span class="spectrum-Badge spectrum-Badge--active">Active</span>
+      <span class="spectrum-Badge spectrum-Badge--inactive">Inactive</span>
+  - id: badge
+    name: Small
+    description: A small badge.
+    markup: |
+      <span class="spectrum-Badge spectrum-Badge--small spectrum-Badge--grey">Grey Badge</span>
```
Comment (lazd) [inline components/badge/metadata/badge.yml:24]: These need to use `sizeS` and friends with the new CSS you pushed, I'll fix

File: tools/component-builder/package.json [hunk 3]
```diff
@@ -33,8 +33,8 @@
     "postcss-calc": "^6.0.0",
     "postcss-dir-pseudo-class": "^5.0.0",
     "postcss-discard-empty": "^4.0.1",
-    "postcss-dropdupedvars": "^1.1.1",
-    "postcss-dropunusedvars": "^1.2.0",
+    "postcss-dropdupedvars": "^1.1.2-alpha.0",
+    "postcss-dropunusedvars": "^1.2.1-alpha.0",
```
Comment (snowystinger) [inline tools/component-builder/package.json:37]: fyi, you're missing postcss-droproot

**Review by snowystinger — COMMENTED** [review]
fyi, checkbox dash is not properly sized
Screen Shot 2021-06-25 at 9 55 40 AM

**Comment by snowystinger** [comment]
It looks like static colors are being used sometimes when they should just be the non-static
```
  --spectrum-semantic-cta-background-color-default: var(--spectrum-global-color-static-blue-600);
  --spectrum-semantic-cta-background-color-hover: var(--spectrum-global-color-static-blue-700);
  --spectrum-semantic-cta-background-color-down: var(--spectrum-global-color-static-blue-800);
  --spectrum-semantic-cta-background-color-key-focus: var(--spectrum-global-color-static-blue-600);
```
This is currently making it so that an emphasized checkbox in dark theme is initially too dark, and then as hover, active, selected all happen, each gets darker when they should be getting lighter.

File: components/vars/css/components/spectrum-checkbox.css [hunk 4]
```diff
@@ -1,1084 +1,2284 @@
...
+  --spectrum-checkbox-m-box-margin-bottom: 0;
+  --spectrum-checkbox-m-box-border-radius: var(--spectrum-alias-border-radius-small);
+  --spectrum-checkbox-m-box-border-size: var(--spectrum-alias-border-size-thick);
+  --spectrum-checkbox-m-box-padding: 0;
+  --spectrum-checkbox-m-box-gap: var(--spectrum-global-dimension-size-125);
+  --spectrum-checkbox-m-box-size: var(--spectrum-global-dimension-size-175);
+  --spectrum-checkbox-m-text-size: var(--spectrum-global-dimension-font-size-100);
+  --spectrum-checkbox-m-text-gap: var(--spectrum-global-dimension-size-125);
+  --spectrum-checkbox-m-text-transform: none;
+  --spectrum-checkbox-m-text-padding-top: var(--spectrum-global-dimension-size-75);
```
Comment (snowystinger) [inline components/vars/css/components/spectrum-checkbox.css:1121]: I think this is supposed to be 65?

**Pull request merged**